BUILD

```diff
@@ -8,3 +9,8 @@ license(
 licenses(["notice"])
 
 exports_files(["LICENSE"])
+
+proto_toolchain(
```

Comment:
@thesayyn didn't know where to place the call to the toolchain. I am not familiar with bzlmod and module extensions. Do you know if its possible to define the toolchain outside BUILD.bazel?

Reply:
Just found some related work in https://github.com/bazelbuild/rules_proto/pull/206/files It's a big chunk.

Reply:
Tried it in https://github.com/psalaberria002/rules_proto/pull/1 but it seems overkill.